src/AUDIT_TRAIL_2.md

```diff
@@ -0,0 +1,49 @@
+# Audit trail for Mangrove v2
```

Comment:
mention bit-masking for makers along with read-lock?

Reply:
I think this is addressed by copy/pasting Espen's pasted section, but let me know if not.